test/integration/run_gadgetron_test.py

```diff
@@ -111,6 +112,16 @@ def send_data_to_gadgetron(echo_handler, gadgetron, *, input, output, configurat
                    stderr=log)
 
 
+def wait_for_server(port, retries=20):
+    with socket.socket(socket.AF_INET) as sock:
+
+        for i in range(retries):
+            if sock.connect_ex("localhost", port) == 0:
```

Comment:
I get an error from this line when trying to run this. I have not dug in, but it also fails on the build agent.